Eluant/LuaRuntime.cs

```diff
@@ -492,9 +493,9 @@ private void ProcessReleasedReferences()
             }
         }
 
-        private void LoadString(string str)
+        private void LoadBuffer(string str, string name)
```

Comment:
This was to match the inner `luaL_loadstring` &rarr; `luaL_loadbuffer` change.  It is better to look at this as the addition of a new `LoadBuffer` method and then the removal of the redundant (for our purposes) `LoadString`.  I agree that we should keep both `LoadBuffer` and `LoadString` before you merge this upstream.